app/views/root/signin_required.html.erb

```diff
@@ -1,16 +1,24 @@
-<% content_for :title, "Sorry, you don't have permission to access " + application_name %>
+<% content_for :title, title %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <% if gds_only_application_and_non_gds_user? %>
+    <% else %>
```

Comment:
Is the above part of the `if` meant to be empty?

Reply:
Yes, fixed in 9cae5e4b1cbe9786deb7b44c7a594904900ef85a